eoscostarica/eos-rate
# refactor(ci): remove gulp dependency and replace with makefile

### Steps to test
1. Run `make setup` if you haven't run the `setup` task before. If you already have, you can use `make start` to start dev environment. It should just work.
2. Look for any instance of `gulp`, there shouldn't be one.
